src/index.d.ts

```diff
@@ -4655,10 +4655,22 @@ declare namespace esb {
              * Missing specifies the value to use when the source finds a missing value
              * in a document.
              *
+             * Note: The `missing` option of the composite aggregation is deprecated in
+             * [Elastticsearch v6.0](https://www.elastic.co/guide/en/elasticsearch/reference/6.8/breaking-changes-6.0.html#_literal_missing_literal_is_deprecated_in_the_literal_composite_literal_aggregation),
```

Comment:
```suggestion
             * [Elasticsearch v6](https://www.elastic.co/guide/en/elasticsearch/reference/6.8/breaking-changes-6.0.html#_literal_missing_literal_is_deprecated_in_the_literal_composite_literal_aggregation),
```

Reply:
Addressed in https://github.com/sudo-suhas/elastic-builder/pull/108/commits/96fc5174f33a364a69dbf914da499b1f57cff1de